packages/fetchai/contracts/erc1155/contract.py

```diff
@@ -56,18 +57,20 @@ def __init__(
         super().__init__(contract_id, contract_config, contract_interface)
         self.is_items_created = False
         self.is_items_minted = False
-        self.token_ids = []  # type: List[int]
+        self.is_trade = False
```

Comment:
What is this for?

Reply:
For handling the sequence in the Transaction handler of the deployer.

Reply:
So then it should not be here. It should be in the skill imo.